quickstarts/springboot-with-s2i.yaml

```
@@ -0,0 +1,186 @@
+  introduction: >-
+    **Spring** is a Java framework for building applications based on a distributed microservices architecture. 
```

```suggestion
    **Spring Boot** is a Java framework for building applications based on a distributed microservices architecture. 
```
Also, you might want to check requirements when it comes to trademarks…